libs-scala/nonempty/src/test/scala/nonempty/NonEmptySpec.scala

```diff
@@ -277,6 +277,20 @@ class NonEmptySpec extends AnyWordSpec with Matchers with WordSpecCheckLaws {
     }
   }
 
+  "minBy" should {
+    "'work' on sets, so to speak" in {
```

Comment:
those descriptions look a bit weird but seems consistent with the rest so :shrug: